pilot/pkg/model/authentication.go

```diff
@@ -28,6 +28,14 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+const (
+	// SDSCertPath is the path of cert that envoy uses to communicate with SDS service through secure gPRC.
+	SDSCertPath = "/etc/istio/nodeagent-sds-cert.pem"
```

Comment:
I remember we use to use 'etc/certs' path, any reason to change it?

Reply:
@incfly, who add this cert to docker image originally

Reply:
I don't have strong opinion on paths. Feel free to adjust `Dockerfile.proxyv2` if needed.